### PR TITLE
OCPBUGS#1823: Default pids-limit and log-max-size deprecation note

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OCPBUGS-1823

Link to docs preview: https://52607--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-deprecated-features-crio-parameters

QE review:
- [x] QE has approved this change.

Additional information: CM needed

### CODE REVIEW

**Comment by umohnani8** [comment]
LGTM

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1316,6 +1316,11 @@ In {product-title} 4.11, support for VMware vSphere 7.0 Update 1 or earlier is d
 ==== Support for ESXi 7.0 Update 1 or earlier is deprecated
 In {product-title} 4.11, support for VMware ESXi 7.0 Update 1 or earlier is deprecated. While EXSi 7.0 Update 1 or earlier remains fully supported, Red Hat recommends that you use ESXi 7.0 Update 2 or later.
 
+[id="ocp-4-11-deprecated-features-crio-parameters"]
+==== Support for `pidsLimit` and `logSizeMax` CRI-O parameters will be deprecated
+
+In {product-title} 4.11, support for the `pidsLimit` and `logSizeMax` CRI-O parameters is expected to be deprecated in a future release. As part of the deprecation, the number of PIDs that can be created in a container or pod is no longer limited. It is recommended to use the `PodPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead.
```
Comment (haircommander) [inline release_notes/ocp-4-11-release-notes.adoc:1322]: > As part of the deprecation, the number of PIDs that can be created in a container or pod is no longer limited.

this is not true. While CRI-O is no longer limiting the number of PIDs, there is a KubeletConfig option which does, and https://github.com/openshift/machine-config-operator/pull/3085 applies this by default

Comment (haircommander) [reply release_notes/ocp-4-11-release-notes.adoc:1322]: and I did double check: 527e2a5f56e354e9166e41475620059c71a4e6c2 made it into the release-4.11 branch

**Pull request merged**